doc/Makefile.am

```diff
@@ -82,10 +82,10 @@ man-page-build: upg-page-build man-page-start
 	##
 	for FILE in $${MAN_FILES} ; do \
 		BASE=`basename $${FILE} | cut -d. -f 1` ;\
-		MANUAL=`egrep -B 1 "^====" $${FILE} | head -1` ;\
-		SUMMARY=`egrep -B 2 "^SYNOPSIS" $${FILE} | ${SED} 's/coap-//g' | cut -d\- -f2 | cut -c2- | head -1` ;\
+		MANUAL=`grep -E -B 1 "^====" $${FILE} | head -1` ;\
+		SUMMARY=`grep -E -B 2 "^SYNOPSIS" $${FILE} | ${SED} 's/coap-//g' | cut -d\- -f2 | cut -c2- | head -1` ;\
```

Comment:
The right answer for Makefile.am files is to use `${EGREP}` or preferably now that egrep is deprecated `${GREP} -E` here.

Reply:
Thanks. Changed!